features/config_strict_testers.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Strict result interpretation defined in configuration file
```

Comment:
Renamed the feature file as `config_strict` was slightly ambiguous about *what* is strict.